### ISSUE TITLE
CraftTweaker integration - can't remove recipes that use IIngredients

### ISSUE THREAD

**Issue by luke-bravenboer** [TheLMiffy1111/EssentialCraft4Unofficial#45]
While trying to modify the recipe for the Magical Plate, I was having trouble removing the one that uses the Thaumium ore dict entry; wondering if I had the syntax wrong, I looked into your crafttweaker integration code & found an issue- we can't actually remove any recipes that involve IIngredients.

The MagicianTable.removeRecipe method (and others, too) rely on your CraftTweakerUtils.java's 'getIItemStack()' method- which only deals with IItemStacks, not IIngredients- and so fails to work for any recipe that uses IIngredients, such as the recipe for the Magical Plate:

 [<ore:plateFortified>, <ore:ingotThaumium>, <ore:ingotThaumium>, <ore:ingotThaumium>, <ore:ingotThaumium>]

Basically every removeRecipe() method in the src/main/java/essentialcraft/integration/crafttweaker/ package seems to be affected.

**Comment by TheLMiffy1111** [comment]
The issue doesn't seem to be there.
